Marlin/pins_GT2560.h

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
This board is completely identical to `BOARD_ULTIMAKER`. If it's not a unique board, then you should just use this in `Configuration.h`:
```cpp
#define MOTHERBOARD BOARD_ULTIMAKER
```
Or, if this is a unique board which happens to use all the same pin assignments as ULTIMAKER, then this file shoud contain only the following:
```cpp
/**
 * GT2560 pin assignments
 */
#include "pins_ULTIMAKER.h"

#undef DEFAULT_MACHINE_NAME
#define DEFAULT_MACHINE_NAME    "GT2560"
#undef DEFAULT_SOURCE_CODE_URL
```